GoogleForCreators/web-stories-wp
# Design System: Update checkmark on toggle

## Context

Toggle was using the incorrect checkmark.

## Summary

- Use checkmark xs in the toggle. Fix small styling problems

|Before|After|
|--|--|
|![image](https://user-images.githubusercontent.com/22185279/111680527-eee34900-87e7-11eb-86e4-a313f022789d.png)|![image](https://user-images.githubusercontent.com/22185279/111680501-e8ed6800-87e7-11eb-8578-c42e91872f28.png)|

## Relevant Technical Choices

none

## To-do

none

## User-facing changes

|Before|After|
|--|--|
|![image](https://user-images.githubusercontent.com/22185279/111681073-7e88f780-87e8-11eb-82cc-e297b4ef4e86.png)|![image](https://user-images.githubusercontent.com/22185279/111680837-3bc71f80-87e8-11eb-82f2-1df86239dccd.png)|


## Testing Instructions

1. Go to storybook. Design System -> toggle
2.  Look at toggle styles

### QA






1. Go to editor
2. Go to text sets in the library panel
3. Click toggle. Should see new checkmark icon very slightly different.

### UAT






1. Go to storybook. Design System -> toggle
2.  Look at toggle styles

## Reviews

### Does this PR have a security-related impact?

no

### Does this PR change what data or activity we track or use?

no

### Does this PR have a legal-related impact?

no

## Checklist




---



Fixes #6691
